chalice/app.py

```diff
@@ -17,6 +18,13 @@
 _PARAMS = re.compile(r'{\w+}')
 
 
+def base64decode(encoded):
```

Comment:
I only see this being used in one spot.  I'd prefer to just move it into the class that's using it.